app/services/nomis/models/sentence_detail.rb

```diff
@@ -14,6 +14,18 @@ class SentenceDetail
       attribute :facial_image_id
       attribute :internal_location_desc
 
+      def earliest_release_date
```

Comment:
This may be a better place for this than Offender or OffenderShort, we should remember to see if we can either move it out of there, or store it somehow.

Reply:
I'll add a ticket so we can come back to this